pkg/kubelet/kubelet.go

```diff
@@ -929,14 +957,8 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		kubeDeps.Recorder,
 		volumepathhandler.NewBlockVolumePathHandler())
 
-	boMax := MaxContainerBackOff
-	base := containerBackOffPeriod
-	if utilfeature.DefaultFeatureGate.Enabled(features.KubeletCrashLoopBackOffMax) {
-		boMax = kubeCfg.CrashLoopBackOff.MaxContainerRestartPeriod.Duration
-		if boMax < containerBackOffPeriod {
-			base = boMax
-		}
-	}
+	boMax, base := MakeCrashLoopBackOffDecay(kubeCfg)
+
 	klet.backOff = flowcontrol.NewBackOff(base, boMax)
```

Comment:
This is only ever used for crashloop backoff, right? Can you rename it to `crashloopBackoff`?